### PR TITLE
Updated for InterPro 101.0

### PR DESCRIPTION
**Pull request by blazaropinto**
Updates

### CODE REVIEW

**Review by typhainepl — CHANGES_REQUESTED** [review]
Good job! I made a few corrections to the text, but nothing major.
1 image needs to be updated

File: docs/images/search/browse_page.png
[the remark this answers is not in the document]
Comment (blazaropinto) [reply docs/images/search/browse_page.png:1]: Thank you Typhaine! I updated the image so it should be ready now.

[the remark this answers is not in the document]
Comment (typhainepl) [reply docs/images/search/browse_page.png:1]: looking good now

**Review by typhainepl — APPROVED** [review]
Feel free to merge into master.

**Pull request merged**